unstructured/ingest/evaluate.py

```diff
@@ -123,6 +132,73 @@ def measure_edit_distance(
     _display(agg_rows, headers)
 
 
+@main.command()
+@click.option("--output_dir", type=click.STRING, help="Directory to a structured output.")
+@click.option(
+    "--output_list",
+    type=click.STRING,
+    multiple=True,
+    help="Optional: list of selected structured output file names under the \
+        directory to be evaluate. If none, all files under directory will be use.",
+)
+@click.option("--source_dir", type=click.STRING, help="Directory to a structured source.")
+@click.option(
+    "--source_list",
+    type=click.STRING,
+    multiple=True,
+    help="Optional: list of selected structured source file names under the directory \
+        to be evaluate. If none, all files under directory will be use.",
+)
+@click.option(
+    "--export_dir",
+    type=click.STRING,
+    default="metrics",
+    help="Directory to save the output evaluation metrics to. Default to \
+        [your_working_dir]/metrics/",
+)
+def measure_element_type_accuracy(
+    output_dir: str,
+    output_list: Optional[List[str]],
+    source_dir: str,
+    source_list: Optional[List[str]],
+    export_dir: str,
+):
+    if not output_list:
+        output_list = _listdir_recursive(output_dir)
+    if not source_list:
+        source_list = _listdir_recursive(source_dir)
+
+    rows = []
+    accuracy_scores: List[float] = []
+
+    for doc in output_list:  # type: ignore
+        fn = (doc.split("/")[-1]).split(".json")[0]
+        connector = doc.split("/")[0]
+        if doc in source_list:  # type: ignore
+            output = get_element_type_frequency(_read_json(os.path.join(output_dir, doc)))
+            source = get_element_type_frequency(_read_json(os.path.join(source_dir, doc)))
+            accuracy = round(calculate_element_type_percent_match(output, source), 3)
```

Comment:
should the result rounding be built into the `calculate_element_type_percent_match` function directly?

Reply:
I think it is ok to leave it here in case we or the user wants more precision.

Reply:
hm, that's ok too, but I was imagining it could be an input ("rounding") to that calculate_element_type_percent_match function with a default value of say 3, so it can always be changed or removed for more precision there if desired. maybe we can make a different task / ticket to move this later